src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -608,7 +613,8 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
         }
 
         internal static async Task InvokeAsync(IFunctionInvoker invoker, object[] invokeParameters, CancellationTokenSource timeoutTokenSource,
-            CancellationTokenSource functionCancellationTokenSource, bool throwOnTimeout, TimeSpan timerInterval, IFunctionInstance instance)
+            CancellationTokenSource functionCancellationTokenSource, bool throwOnTimeout, TimeSpan timerInterval, IFunctionInstance instance, ILogger logger = null,
+            JobHost jobHost = null, JobHostConfiguration config = null, IReadOnlyList<string> parameterNames = null)
```

Comment:
The reason behind putting the "= null" was because when I would build it, certain functions in other files would call the function and not provide the new parameters thus causing errors. I can remove the "= null" but then I'd have to pass the new parameters in other places in the project where it does call it. I'd also have to modify other tests.